plugins/modules/launchdarkly_feature_flag_sync.py

```diff
@@ -13,49 +13,49 @@
 DOCUMENTATION = r"""
 ---
 module: launchdarkly_feature_flag_sync
-short_description: Sync LaunchDarkly Feature Flags across Environments
+short_description: Sync flag settings across environments
 description:
-     - Sync LaunchDarkly Feature Flags across Environments
+     - Sync LaunchDarkly feature flag settings across environments. To learn more, read L(Comparing and copying flag settings between two environments, https://docs.launchdarkly.com/home/code/flag-compare-copy#comparing-and-copying-flag-settings-between-two-environments).
 version_added: "0.1.0"
 options:
     project_key:
         description:
-            - Project key to look for flag
+            - The project key
         default: 'default'
         type: str
     flag_key:
         description:
-            - A unique key that will be used to reference the user segment in this environment.
+            - The flag key
         type: str
         required: yes
     environment_key:
         description:
-            - A unique key that will be used to determine source environment.
+            - The envionment key for the source environment
         required: yes
         type: str
     environment_targets:
         description:
-            - A list of environments that flag settings will be copied to.
+            - A list of environment keys that flag settings will be copied to
         required: yes
         type: list
     included_actions:
         description:
-            - Manage a list of included actions for copying. If not specified all actions are included.
+            - Manage a list of included actions to copy. If not specified, all actions are included.
         required: no
         type: list
-        choices: ['updateOn', 'updatePrerequisites', 'updateTargets', 'updateRules', 'updateFallthrough', 'updateOffVariation']
+        choices: ['updateFallthrough', 'updateOn', 'updateOffVariation', 'updatePrerequisites', 'updateRules', 'updateTargets']
```

Comment:
Reordered to be alphabetical. Existing REST API docs have a different order, so I'm pretty certain the order doesn't matter here, but please correct me if that's not the case.

Reply:
Nope, order doesn't matter.